src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -36,6 +38,10 @@ export class SchoolSelectField extends React.Component {
       150);
   }
 
+  componentWillUnmount() {
+    this.debouncedSearchInstitutions.cancel();
```

Comment:
Addresses [this](https://github.com/department-of-veterans-affairs/vets-website/pull/8062#discussion_r206729036).